packages/expo-dev-launcher/plugin/src/withDevLauncher.ts

```diff
@@ -13,14 +13,14 @@ export default createRunOncePlugin<PluginConfigType>(
   (config, props) => {
     validateConfig(props || {});
 
-    if (props.ios?.tryToLaunchLastOpenedBundle ?? props.tryToLaunchLastOpenedBundle) {
+    if (props?.ios?.tryToLaunchLastOpenedBundle ?? props?.tryToLaunchLastOpenedBundle ?? false) {
```

Comment:
My bad, fixed here https://github.com/expo/expo/pull/24646

Reply:
Thank you!